### DISCUSSION TITLE
How do I ensure that only relevant deps in node_modules are installed in docker

### DISCUSSION THREAD

**Discussion by Yuripetusko** [vercel/turborepo#4317] [Help] [answered]
Trying to understand how do I ensure that only relevant deps end up in docker's node_modules, since I am copying root package.json and well as app's package.json when installing deps and running docker build, most of my packages end up in root workspace node_modules, which means I then need to copy over the whole root node_modules to docker container that contains deps that are not necessarily needed for this particular app.

**Comment by anthonyshew** [comment, marked as answer]
Check out a couple of our docs:
- `turbo prune`: https://turbo.build/repo/docs/reference/command-line-reference/prune
- Deploying with Docker: https://turbo.build/repo/docs/handbook/deploying-with-docker
